packages/apps/transfer/package.json

```diff
@@ -21,9 +21,11 @@
     "@kadena/chainweb-node-client": "workspace:*",
     "@kadena/client": "workspace:*",
     "@kadena/cryptography-utils": "workspace:*",
+    "@kadena/kadena-docs": "workspace:*",
```

Comment:
Do we need/use this dependency (app in app)?

Reply:
nope, i was trying to get docs component, thanks for reminding me, removed